src/bidiMapper/bidiProtocolTypes.ts

```diff
@@ -212,6 +213,11 @@ export namespace CommonDataTypes {
     value?: NodeProperties;
   };
 
+  export type LogRemoteValue = RemoteReference & {
```

Comment:
This type isn't a part of the spec, and the args may not always be strings. Please update the code below to use RemoteValue.

Reply:
Without a LogRemoteValue type, we will get a build error. There is no existing RemoteValue type that matches {type: string, value: string}.

Reply:
Removed Log.RemoteValue.